README.md

```diff
@@ -1,51 +1,394 @@
-# HLASM Language Support
-[![CI](https://github.com/eclipse/che-che4z-lsp-for-hlasm/workflows/CI/badge.svg?branch=development)](https://github.com/eclipse/che-che4z-lsp-for-hlasm/actions?query=workflow%3ACI)
+<div id="header" align="center">
+
 [![GitHub issues](https://img.shields.io/github/issues-raw/eclipse/che-che4z-lsp-for-hlasm)](https://github.com/eclipse/che-che4z-lsp-for-hlasm/issues)
 [![slack](https://img.shields.io/badge/chat-on%20Slack-blue)](https://communityinviter.com/apps/che4z/code4z)
 <a href="https://sonarcloud.io/dashboard?id=eclipse_che-che4z-lsp-for-hlasm">
-<img src="https://sonarcloud.io/images/project_badges/sonarcloud-black.svg" width="94" height="20" href="" />
+<img src="https://github.com/eclipse/che-che4z-lsp-for-hlasm/raw/master/clients/vscode-hlasmplugin/sonarcloud-black.png" width="94" height="20" href="" />
```

Comment:
I have no clue why was the logo included in the repo to begin with, but were various copyright implications investigated back then?

Reply:
More pressingly the image was of poor resolution so I've replaced it with the badge used on COBOL LS. @zimlu02 are we allowed to use the SonarCloud logo on that badge? :)